NEWS.md

```diff
@@ -162,6 +162,9 @@ fail.
 * `stat_summary()` and related functions now support rlang-style lambda functions
   (#3568, @dkahle).
 
+* `strip_dots()` strips the data masking pronoun `.data` to make better
```

Comment:
`strip_dots()` is not an exported function, so I think it might better to write more simply like "The data mask pronoun `.data` is now stripped from default labels."

Reply:
Good idea, thanks!